src/shared-components/dropdown/index.tsx

```diff
@@ -125,35 +77,16 @@ export const Dropdown = <T extends OptionType>({
     }
   };
 
-  if (touchSupported) {
-    return (
-      <MobileDropdown
-        borderRadius={borderRadiusValue}
-        id={id}
-        onMobileSelectChange={onMobileSelectChange}
-        onDropdownContainerFocus={handleOnDropdownContainerFocus}
-        options={options}
-        textAlign={textAlign}
-        value={value}
-      />
-    );
-  }
-
-  const currentOption = options.find((option) => option.value === value);
-
+  // if (touchSupported) {}
   return (
-    <DesktopDropdown
+    <GenericConfigurableDropdown
```

Comment:
Since we've done away with the mobile and desktop components, do you think it might cause less indirection to move the logic contained in `GenericConfigurableDropdown` into this `Dropdown` component?

Reply:
I think this is a good suggestion. I was a bit worried about condensing it all down into that top-level component in one go because I am still pretty new and having two modules felt like it would allow for cleaner context-based configuration in the future, but this pattern doesn't really buy us anything right now